CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 36.0.0 [#1160](https://github.com/openfisca/openfisca-core/pull/1160)
+
+#### Breaking changes
+
+- This pull request changes the definition of what is to be considered as a breaking change in a non-retro-compatible way.
+- Before, a breaking change in core automatically forced a breaking change in all it's dependants.
+- Now, a breaking change in core is a breaking change in any dependant relying on actually deprecated features and not just a major version bump in core.  
```

Comment:
Why is this a breaking change for reusers? 🙂 In my view this is a minor bump with new feature: lighter install by removing test packages from systematic install.